<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="adminpdfsAir73" ignored="2" name="Login" tests="5" failures="1" timestamp="16 Jul 2019 13:20:14 GMT" time="26.838" errors="0">
  <testcase name="login" time="12.846" classname="regression.LoginTest"/>
  <testcase name="login" time="4.607" classname="regression.CardActions"/>
  <testcase name="moveCard" time="0.002" classname="regression.CardActions"/>
  <testcase name="openCard" time="2.003" classname="regression.CardActions">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at regression.CardActions.openCard(CardActions.java:63)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- openCard -->
  <testcase name="renameCard" time="0.059" classname="regression.CardActions"/>
  <testcase name="@AfterTest tearDown" time="0.0" classname="com.trello.ui.core.BrowserFactory">
    <ignored/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="@BeforeTest setUp" time="0.0" classname="com.trello.ui.core.BrowserFactory">
    <ignored/>
  </testcase> <!-- @BeforeTest setUp -->
</testsuite> <!-- Login -->
